src/components/implementation/no_interface/vkernel/vk_server.c

```diff
@@ -0,0 +1,47 @@
+
+#include "vk_types.h"
+
+int
+vkernel_find_vm(thdid_t tid)
+{
+	int i;
+
+	for (i = 0 ; i < VM_COUNT ; i ++) {
+		if ((vmx_info[i].inithd)->thdid == tid) break;
+	}
+	assert (i < VM_COUNT);
+
+	return i;
+}
+
+int
+vkernel_server(int a, int b, int c)
```

Comment:
`server` is a noun which is strange for an execution entry point.

`vkernel_hypercall` might better convey what is going on here.